Packs/RubrikPolaris/Integrations/RubrikPolaris/RubrikPolaris_description.md

```diff
@@ -1,15 +1,10 @@
-Support and maintenance for this integration are provided by the author. 
-Please use the following contact details:
-- "Email": [support@rubrik.com](mailto:support@rubrik.com)
-- "URL": [https://www.rubrik.com/support/](https://www.rubrik.com/support/)
-
```

Comment:
Why remove?

Reply:
This was dupliacte and already coming from pack metadata directly.